### DISCUSSION TITLE
Feature Request - Wiki

### DISCUSSION THREAD

**Discussion by 1sc** [community/community#62] [Mobile] [answered]
It would be great to have access to wiki inside of the app.

**Comment by candyho** [comment, marked as answer]
Hey thank you for the feedback. Wiki access on mobile is on our backlog. Will keep you updated when we can land a solution for this.

**Reply by drudge** [reply]
Hoping this is still going to happen.

**Reply by iqbalrezaamali** [reply]
[Feature Request - Wiki](https://github.com/orgs/community/discussions/62)Feature Request - Wiki

**Comment by Wladefant** [comment]
Duplicate of #3635

**Reply by iqbalrezaamali** [reply]
![Uploading Screenshot_2023-03-13-23-05-10-69_934c6d2db9b5f7f372f944051cd81fb0.jpg…]()

**Comment by iqbalrezaamali** [comment]
#49932
